pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -298,6 +311,18 @@ func (s *NodeController) DoCheck(node *api.Node) []api.NodeCondition {
 	return conditions
 }
 
+// SendConditionEvent sends condition change event.
+func (s *NodeController) SendConditionEvent(node *api.Node, kind api.NodeConditionType, oldCondition, newCondition *api.NodeCondition) {
+	// If 'oldCondition' is nil, node is checked for the first time, we have already sent
+	// created event. Plus it's impossible to send event since node has no SelfLink here.
+	if oldCondition != nil {
+		condKind := strings.ToLower(string(kind))
+		oldStatus := strings.ToLower(string(oldCondition.Status))
+		newStatus := strings.ToLower(string(newCondition.Status))
```

Comment:
Thoughts here:
* Our reasons are usually camelCase, but this will make alllowercase.
* When people search the codebase for one of the strings this generates, they won't find anything.
* Is it useful to have oldStatus as part of the event reason? Why not just becameNewStatus?